ding_dibs.install

```diff
@@ -1,6 +1,43 @@
 <?php
 
 /**
+ * Implementation of hook_schema().
+ */
+function ding_dibs_schema() {
+  $schema['ding_dibs_report_cron'] = array(
+    'description' => 'Report cron table.',
+    'fields' => array(
+      'id' => array(
+        'type' => 'serial',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+      ),
+      'time_start' => array(
+        'type' => 'varchar',
+        'length' => 32,
```

Comment:
Why are these time fields defined as varchars? I would expect an int field for timestamps.